### PR TITLE
Great places to eat in Mtn View CA

### PR DESCRIPTION
**Pull request by Estherbunny**
Know any others?

### CODE REVIEW

File: mountainviewca-eats.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Places to eat in Mountain View, CA
+
+- Bierhaus
+- Stein's
+- Agave
```
Comment (ebuceci) [inline mountainviewca-eats.md]: Haven't been to Agave yet! One of these days...!

**Comment by ntend** [comment]
@Estherbunny How about Cascal? Have you eaten there?

**Comment by Estherbunny** [comment]
@ntend yes! we usually avoid it because it gets so crowded, but the food is spot on - adding

File: mountainviewca-eats.md (hunk 1 above)
Comment (Estherbunny) [inline mountainviewca-eats.md]: we love it- great food and atmosphere, and tons of outdoor seating in the back

**Pull request merged**